src/tests/accommodation/accommodationFeedback.test.js

```diff
@@ -0,0 +1,64 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import sinonChai from 'sinon-chai';
+import { mockReq, mockRes } from 'sinon-express-mock';
+import localStorage from 'localStorage';
+import sinon from 'sinon';
+import app from '../../app';
+import accommodationFacilities from '../../controllers/accommodation';
+import { travelToken } from './accommodationMockData';
+
+chai.use(chaiHttp);
+chai.should();
+chai.use(sinonChai);
+const accommodationFeedBack = () => {
+  describe('accommodation feedback ', () => {
+    before((done) => {
+      localStorage.setItem('token', travelToken);
+      done();
+    });
+    it('it should return 200 when accommodation commented successfully ', () => {
+      const accommodationFeedBackSpy = sinon.spy(accommodationFacilities, 'accommodationFeedBack');
+      const request = {
+        params: {
+          id: 1,
+        },
+        body: {
+          coment: 'You have good servise',
+        },
+        user: {
+          id: 6,
+        },
+      };
+      const req = mockReq(request);
+      const res = mockRes();
+      accommodationFacilities.accommodationFeedBack(req, res);
+      expect(accommodationFeedBackSpy).to.have.been.calledWith(req, res);
+      accommodationFeedBackSpy.restore();
+    });
+    it('it should return 200 when you commented ', (done) => {
+      chai
+        .request(app)
+        .post('/api/v1/accommodation/comment/1')
+        .send({ comment: 'Bad service' })
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(201);
```

Comment:
You should expect not expect the status only.

Reply:
cool let me work on it

Reply:
I have tested everything you can check